### DISCUSSION TITLE
[Android] The image is not displayed after upgrading to uno4.5

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/uno#9864] [Q&A] [answered]
The image is not displayed after upgrading to uno4.5. Such as:
`<Image Source="ms-appx:///Assets/logo.png" Stretch="None" />`

You must modify the `Source` to
`<Image Source="ms-appx:///logo.png" Stretch="None" />`
or
`<Image Source="logo.png" Stretch="None" />`

This makes it impossible to unify the format across platforms.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. Can you create a repro sample that demonstrates the problem?

**Reply by Daoting** [reply]
I still use the original image asset method because I don't use a shared library.
![image](https://user-images.githubusercontent.com/29876815/190941548-cf0ea036-f98d-412d-9ba9-3fb30c9b2f7e.png)

Now I need to put the image in the Assets directory and use it as content, and it can be displayed.
![image](https://user-images.githubusercontent.com/29876815/190942336-c9827448-97a8-43f9-824f-7478673da42a.png)


Thanks.
